index.js

```diff
@@ -189,7 +190,9 @@ var html = fs.readFileSync(path.join(__dirname, 'tree-viz.html')).toString();
 
 html = html.replace('INSERT TREE HERE', JSON.stringify(sizes, null, '  '))
            .replace('INSERT TITLE HERE', args['<script.js>'])
-
+           .replace('underscore.js', fileURL(require.resolve('underscore')))
+           .replace('webtreemap.js', fileURL(require.resolve('./vendor/webtreemap.js')))
```

Comment:
I wrote it on Windows :smile_cat: Node.js path stuff works fine on Windows IME if you just use `/` everywhere, as long as the developers of libs you're using `path`.